### PR TITLE
Update cm-peri-sw-guide.md

### PR DESCRIPTION
**Pull request by RogueM**

### CODE REVIEW

**Comment by RogueM** [comment]
diff is not very clear for some reasons... so to avoid you parsing it wondering what changed, it is in 'and all the product-specific information contained in a separate overlay.' (typo and language).

**Comment by JamesH65** [comment]
Technical check OK, pushing to copy edit for final proofing @AisforAstronaut

**Comment by AisforAstronaut** [comment]
Can you link to the document? I can't see it on the list.

Thank you!

**Comment by AisforAstronaut** [comment]
Checked and altered. I hadn't seen this document before, so I've done a
full edit. I've restructured some of the very long sentences, so it
probably needs a final technical check.

File: hardware/computemodule/cm-peri-sw-guide.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
 
-To use the Compute Module, a user needs to design a (relatively simple) 'motherboard' which can provide power to the Compute Module (3.3V and 1.8V at minimum) and which wires the pins up to the required peripherals for the user's application.
+To use the Compute Module, a user needs to design a (relatively simple) 'motherboard' which can provide power to the Compute Module (3.3V and 1.8V at minimum,) and which connects the pins to the required peripherals for the user's application.
```
Comment (JamesH65) [inline hardware/computemodule/cm-peri-sw-guide.md]: Extraneous `,` ?

File: hardware/computemodule/cm-peri-sw-guide.md [hunk 2]
```diff
@@ -146,25 +148,27 @@ You can include more diagnostics in the output by adding `dtdebug=1` to `config.
 
 ##Getting Help
 
-Please use the [Device Tree subforum](https://www.raspberrypi.org/forums/viewforum.php?f=107) on the Raspberry Pi forums to ask Device Tree related questions.
+Please use the [Device Tree subforum](https://www.raspberrypi.org/forums/viewforum.php?f=107) on the Raspberry Pi forums to ask Device Tree-related questions.
```
Comment (JamesH65) [inline hardware/computemodule/cm-peri-sw-guide.md]: hyphen seems to be wrong ?

**Comment by AisforAstronaut** [comment]
The first one is a typo - apologies. the comma should be after the bracket,
not before.

The second one I debated for a while. If it looks wrong, get rid of the
hyphen. I think technically it should be 'Device-Tree-related', but that
looks horrible! Ditch the hyphen. Let's pretend this never happened...

:-)

**Comment by JamesH65** [comment]
Fixed up and merged, thanks all.

**Pull request merged**